garb/garb_recv_loop.cpp

```diff
@@ -162,7 +172,16 @@ RecvLoop::loop()
                 if (cc.memb.size() == 0) // SELF-LEAVE after closing connection
                 {
                     if(!config_.recv_script().empty()) {
-                        if(sst_ended) {
+                      if (sst_terminated) {
```

Comment:
It is the same as `else if(sst_ended)`. Is it OK?
Maby we don't need `sst_terminated` as we spoke on Slack?

Reply:
The variable `sst_terminated` is needed, as that's set at a different time (earlier) than `sst_ended`. Otherwise, we end up in the `else` clause here, and call `terminate` again, and that was causing the incorrect return codes.

But you are right, the if blocks can be the same with an or.